src/core/src/op/util/unary_elementwise_arithmetic.cpp

```diff
@@ -12,11 +12,11 @@ ov::op::util::UnaryElementwiseArithmetic::UnaryElementwiseArithmetic(const Outpu
 
 void ov::op::util::UnaryElementwiseArithmetic::validate_and_infer_elementwise_arithmetic() {
     const auto& element_type = get_input_element_type(0);
-    NODE_VALIDATION_CHECK(this,
-                          element_type.is_dynamic() || element_type != element::boolean,
-                          "Arguments cannot have boolean element type (argument element type: ",
-                          element_type,
-                          ").");
+    NODE_VALIDATION_CHECK(
+        this,
+        element_type.is_dynamic() || (element_type != element::boolean && element_type != element::string),
+        "This operation does not support input with element type: ",
+        element_type);
```

Comment:
```suggestion
    const auto is_supported_et = (element_type != element::boolean && element_type != element::string);
    NODE_VALIDATION_CHECK(
        this,
        element_type.is_dynamic() || is_supported_et,
        "This operation does not support input with element type: ",
        element_type);
```
To reduce condition string. The not supported precision will be in exception message.
Same for binary elementwise.

Reply:
Ok done.